rma_sale/views/sale_portal_template.xml

```diff
@@ -92,7 +92,10 @@
                             <th class="text-start">Product</th>
                             <th class="text-end">Quantity</th>
                             <th class="text-start">Delivery</th>
-                            <th class="text-start">Requested operation</th>
+                            <th
+                                class="text-start"
+                                name="operation"
```

Comment:
And you can take the occasion to name all the elements.

```suggestion
                                name="th_operation"
```

Reply:
is it really needed? As in the xpath, we must already specify the node

`<xpath expr="//th[@name='operation']" position="before">`

Reply:
Adding the same name several times is not a good practice IMO. It's true that selectors are done usually with the HTML tag, but to get an absolute element, better with unique names.

Reply:
You are right @pedrobaeza in the case of the `id` element attribute since this one is expected to be unique across the full dom. But for the `name`, it seems to me to be redundant.

Reply:
OK, it's not critical.

Reply:
Just as reference, Odoo follows my same suggestion:

https://github.com/odoo/odoo/blob/9919961ffbc9d05d87f068ee8512728345d17bf1/addons/sale/report/ir_actions_report_templates.xml#L73

Reply:
I would say it should be at developer's discretion as both approaches provide a unique way of xpath anchor.